<launch>
  <arg name="generation_name" default="test"/>
  <arg name="count" default="30"/>
  <arg name="generate_time_interval" default="0"/>

  <node name="bt_prompt" pkg="bt_prompt" type="parse_all.py" output="screen">
    <param name="generation_name" value="$(arg generation_name)" />
    <param name="model_name" value="text-davinci-003" />
    <param name="count" value="$(arg count)"/>
    <param name="generate_time_interval" value="$(arg generate_time_interval)"/>
  </node>

</launch>
